week-03/main.js

```diff
@@ -12,25 +14,40 @@ var list = [1, 2, 3, 4, 5];
 // `isAlive` - a boolean,
 // and `hobbies` an array.
 // Assign the object to the variable `user`
-
+var user = {
+  name: 'kora',
+  isAlive: true,
+  hobbies: ['this', 'that']
+}
 
 
 // Add 'golf' to the `user`'s `hobby` array.
-
+user.hobbies.push('golf');
 
 // # Functions
 
 // Define a function `sayHello` that takes a name string and returns "Hello" plus that name and an exclamation point.
 // Example:
 
 // sayHello('Mer') // "Hello Mer!"
-
+var sayHello = function(name){
+  return 'Hello ' + name + '!';
+}
 // Define a function `createRange` that takes `min` and `max` as parameters and returns an array with all the numbers
 // between min and max
 // Example:
 
 // createRange(4, 8) // [4, 5, 6, 7, 8]
 
+var array = [];
```

Comment:
This array should be declared inside your function, otherwise future calls to `createRange` would add to a global array.